FreeRTOS/Lab-Project-FreeRTOS-FAT
# Fix cmake configure for STM32F4XX and STM32F7XX

Fixed bug that prevented STM32F4XX and STM32F7XX from correctly configure with cmake.

Description
-----------
Setting `FREERTOS_PLUS_FAT_PORT` to STM32F4XX would throw fatal error at line 57 of root CMakeLists.txt since not in `FREERTOS_PLUS_FAT_PORT_LIST`.
Setting `FREERTOS_PLUS_FAT_PORT` to STM32FXX as in the list of ports (notice no number before XX), would throw error since at `portable/CMakeLists.txt:38` it would NOT mach with any generator and result in no sources for `freertos_plus_fat_port` target.

Test Steps
-----------

set `FREERTOS_PLUS_FAT_PORT` to either STM32F4XX or STM32F7XX

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
